### ISSUE TITLE
keep2share error - no Captcha request :(

### ISSUE THREAD

**Issue by kveiro** [pyload/pyload#2573]
Hi,
It seems something has changed on hoster keep2share, i don't get captcha request anymore.
Download is broken :(
Here is my log:

INFO	ADDON UpdateManager: All plugins are up to date!
162	18.08.2016 15:01:21	INFO	Added package http://k2s.cc/file/797cc364cc8d6/Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi containing 1 links
163	18.08.2016 15:01:23	INFO	Download starts: http://k2s.cc/file/797cc364cc8d6/Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi
164	18.08.2016 15:01:23	DEBUG	ADDON ExternalScripts: No script found under folder `download_preparing`
165	18.08.2016 15:01:23	DEBUG	ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
166	18.08.2016 15:01:23	DEBUG	HOSTER Keep2ShareCc[45]: Plugin version: 0.29
167	18.08.2016 15:01:23	DEBUG	HOSTER Keep2ShareCc[45]: Plugin status: testing
168	18.08.2016 15:01:23	WARNING	HOSTER Keep2ShareCc[45]: Plugin may be unstable
169	18.08.2016 15:01:23	DEBUG	Redirected import module.plugins.accounts.Keep2ShareCc -> userplugins.accounts.Keep2ShareCc
170	18.08.2016 15:01:23	DEBUG	Redirected import module.plugins.internal.Account -> userplugins.internal.Account
171	18.08.2016 15:01:23	INFO	HOSTER Keep2ShareCc[45]: Grabbing link info...
172	18.08.2016 15:01:26	DEBUG	HOSTER Keep2ShareCc[45]: Link info: {'status': 2, 'hash': {}, 'name': u'Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi', 'url': u'http://keep2s.cc/file/797cc364cc8d6', 'pattern': {'S': u'550.0 MB', 'ID': u'797cc364cc8d6', 'N': u'Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi'}, 'size': 576716800}
173	18.08.2016 15:01:26	DEBUG	HOSTER Keep2ShareCc[45]: Previous link info: {}
174	18.08.2016 15:01:26	INFO	HOSTER Keep2ShareCc[45]: Link name: Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi
175	18.08.2016 15:01:26	INFO	HOSTER Keep2ShareCc[45]: Link size: 550.00 MiB (576716800 bytes)
176	18.08.2016 15:01:26	INFO	HOSTER Keep2ShareCc[45]: Link status: online
177	18.08.2016 15:01:26	INFO	HOSTER Keep2ShareCc[45]: Processing url: http://k2s.cc/file/797cc364cc8d6/Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi
178	18.08.2016 15:01:26	INFO	HOSTER Keep2ShareCc[45]: Looking for direct download link...
179	18.08.2016 15:01:26	DEBUG	HOSTER Keep2ShareCc[45]: LOAD URL http://keep2s.cc/file/797cc364cc8d6 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={} | ref=True | just_header=True
180	18.08.2016 15:01:28	INFO	HOSTER Keep2ShareCc[45]: Direct download link detected
181	18.08.2016 15:01:28	INFO	HOSTER Keep2ShareCc[45]: Downloading file...
182	18.08.2016 15:01:28	DEBUG	HOSTER Keep2ShareCc[45]: DOWNLOAD URL http://keep2s.cc/file/797cc364cc8d6 | cookies=True | resume=None | get={} | disposition=True | chunks=None | post={} | ref=True
183	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: File saved
184	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: Checking download...
185	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: File is OK
186	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: Checking file (with built-in rules)...
187	18.08.2016 15:01:35	WARNING	HOSTER Keep2ShareCc[45]: Check result: Html file | Waiting 1 minute and retry
188	18.08.2016 15:01:35	DEBUG	HOSTER Keep2ShareCc[45]: WAIT set to timestamp 1471525356.783000 | Previous waitUntil: 0.000000
189	18.08.2016 15:01:35	DEBUG	HOSTER Keep2ShareCc[45]: RECONNECT required | Previous wantReconnect: False
190	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: Waiting 1 minutes...
191	18.08.2016 15:01:35	INFO	HOSTER Keep2ShareCc[45]: Requiring reconnection...
192	18.08.2016 15:01:44	DEBUG	ADDON ExternalScripts: No script found under folder `download_processed`
193	18.08.2016 15:01:44	DEBUG	ADDON ExternalScripts: No script found under folder `package_processed`
194	18.08.2016 15:01:44	INFO	Download aborted: Second.Chance.s01e01.WEBDLRip.NewStudio.TV.avi

Thanks in advance !

**Comment by kveiro** [comment]
Thanks !
Now download goes ok :), BUT:
Once the download is complete, the file "original.file.name.rar.<b>chunk0"</b> is not renamed to "original.file.name.<b>rar"</b>.
I have two files (chunks and chunk0) in download directory, i have renamed the file manually.
Here is my log:
22.08.2016 15:36:02	DEBUG	HOSTER Keep2ShareCc[47]: Plugin version: 0.31
85	22.08.2016 15:36:02	DEBUG	HOSTER Keep2ShareCc[47]: Plugin status: testing
86	22.08.2016 15:36:02	WARNING	HOSTER Keep2ShareCc[47]: Plugin may be unstable
87	22.08.2016 15:36:02	DEBUG	Redirected import module.plugins.accounts.Keep2ShareCc -> userplugins.accounts.Keep2ShareCc
88	22.08.2016 15:36:02	DEBUG	Redirected import module.plugins.internal.Account -> userplugins.internal.Account
89	22.08.2016 15:36:02	INFO	HOSTER Keep2ShareCc[47]: Grabbing link info...
90	22.08.2016 15:36:03	DEBUG	HOSTER Keep2ShareCc[47]: Link info: {'status': 2, 'hash': {}, 'name': 'Easy Context Menu 1.3 Portable.zip', 'url': u'http://k2s.cc/file/1fb5aacb18029', 'pattern': {'S': u'1.4 MB', 'ID': u'1fb5aacb18029', 'N': u'Easy Context Menu 1.3 Portable.zip'}, 'size': 1468006}
91	22.08.2016 15:36:03	DEBUG	HOSTER Keep2ShareCc[47]: Previous link info: {}
92	22.08.2016 15:36:03	INFO	HOSTER Keep2ShareCc[47]: Link name: Easy Context Menu 1.3 Portable.zip
93	22.08.2016 15:36:03	INFO	HOSTER Keep2ShareCc[47]: Link size: 1.40 MiB (1468006 bytes)
94	22.08.2016 15:36:03	INFO	HOSTER Keep2ShareCc[47]: Link status: online
95	22.08.2016 15:36:03	INFO	HOSTER Keep2ShareCc[47]: Processing url: http://k2s.cc/file/1fb5aacb18029/Easy_Context_Menu_1.3_Portable.zip
96	22.08.2016 15:36:03	DEBUG	HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={} | ref=False | just_header=False
97	22.08.2016 15:36:04	INFO	HOSTER Keep2ShareCc[47]: Processing as free download...
98	22.08.2016 15:36:04	DEBUG	HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'slow_id': u'0785ba38b16b1', 'yt0': ''} | ref=True | just_header=False
99	22.08.2016 15:36:05	DEBUG	CAPTCHA Keep2ShareCc[47]: Captcha | LOAD URL http://k2s.cc/file/captcha.html?v=57baff9ee6675 | redirect=True | cookies=True | get={} | req= | decode=False | multipart=False | post={} | ref=False | just_header=False
100	22.08.2016 15:36:05	INFO	CAPTCHA Keep2ShareCc[47]: Captcha | Using OCR to decrypt captcha...
101	22.08.2016 15:36:05	WARNING	CAPTCHA Keep2ShareCc[47]: Captcha | No OCR result
102	22.08.2016 15:36:17	INFO	CAPTCHA Keep2ShareCc[47]: Captcha | Captcha result: `qejrdo`
103	22.08.2016 15:36:17	DEBUG	HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'CaptchaForm[code]': 'qejrdo', 'yt0': '', 'freeDownloadRequest': 1, 'uniqueId': u'0785ba38b16b1', 'free': 1} | ref=True | just_header=False
104	22.08.2016 15:36:17	INFO	CAPTCHA Keep2ShareCc[47]: Captcha | Correct captcha | qejrdo
105	22.08.2016 15:36:17	DEBUG	HOSTER Keep2ShareCc[47]: WAIT set to timestamp 1471873008.927000 | Previous waitUntil: 0.000000
106	22.08.2016 15:36:17	INFO	HOSTER Keep2ShareCc[47]: Waiting 31 seconds...
107	22.08.2016 15:36:49	DEBUG	HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'free': '1', 'uniqueId': u'0785ba38b16b1'} | ref=True | just_header=False
108	22.08.2016 15:36:50	INFO	HOSTER Keep2ShareCc[47]: Downloading file...
109	22.08.2016 15:36:50	DEBUG	HOSTER Keep2ShareCc[47]: DOWNLOAD URL /file/url.html?file=0785ba38b16b1 | cookies=True | resume=None | get={} | disposition=True | chunks=None | post={} | ref=True
110	22.08.2016 15:36:53	DEBUG	Content-Disposition: Easy Context Menu 1.3 Portable.zip filename*=UTF-8Easy%20Context%20Menu%201.3%20Portable.zip
111	22.08.2016 15:37:21	ERROR	HOSTER Keep2ShareCc[47]:
112	22.08.2016 15:37:21	DEBUG	ADDON ExternalScripts: No script found under folder `download_processed`
113	22.08.2016 15:37:21	DEBUG	ADDON ExternalScripts: No script found under folder `package_processed`
114	22.08.2016 15:37:21	WARNING	Download failed: Easy Context Menu 1.3 Portable.zip | IOError 22
115	22.08.2016 15:37:21	DEBUG	ADDON ExternalScripts: No script found under folder `download_failed`
116	22.08.2016 15:37:21	INFO	ADDON UnSkipOnFail: Looking for skipped duplicates of: Easy Context Menu 1.3 Portable.zip (pid:47)
117	22.08.2016 15:37:21	INFO	ADDON UnSkipOnFail: No duplicates found
